src/parquet/util/bpacking.h

```diff
@@ -13,6 +13,8 @@
 #ifndef PARQUET_UTIL_BPACKING_H
 #define PARQUET_UTIL_BPACKING_H
 
+#include <stdexcept>
```

Comment:
Without this, my compiler gcc version 4.9.3 (Ubuntu 4.9.3-8ubuntu2~14.04) throws an error:
/parquet-cpp/src/parquet/util/bpacking.h: In function ‘int parquet::unpack32(const uint32_t*, uint32_t*, int, int)’:
/parquet/parquet-cpp/src/parquet/util/bpacking.h:3317:13: error: ‘runtime_error’ is not a member of ‘std’
       throw std::runtime_error("Unsupported num_bits");